scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf

```diff
@@ -0,0 +1,148 @@
+# Total Employed Persons
+Node: dcid:Count_Person_Employed
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+employment: dcs:Employed
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+sourceScalingFactor: 1000
+
+
+# Population employed by NACE code (A)
+Node: dcid:Count_Person_Employed_NACE/A
```

Comment:
I am still admittedly confused about schema.  Does this need to be split into a separate pull request containing only schema changes?

Reply:
The enums are already submitted. 
https://github.com/datacommonsorg/data/blob/33c245c287eb6585bd12cff763786738f4b6bb78/schema/nace.mcf